### PR TITLE
TE5-T228 Implemented Kotlin Basic

### PR DESCRIPTION
**Pull request by buddhdev-jay**

### CODE REVIEW

File: app/src/main/java/com/example/android_demo_kotlin/Kotlin_Basic.kt [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+
+class Kotlin_Basic : AppCompatActivity() {
+    @SuppressLint("LongLogTag")
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_kotlin_basic)
+
```
Comment (purvesh-d) [inline app/src/main/java/com/example/android_demo_kotlin/Kotlin_Basic.kt]: Remove extra space

Comment (purvesh-d) [reply app/src/main/java/com/example/android_demo_kotlin/Kotlin_Basic.kt]: Check this same mistake in whole file and update it.

Comment (buddhdev-jay) [reply app/src/main/java/com/example/android_demo_kotlin/Kotlin_Basic.kt]: Okk I Updated the File

File: app/src/main/java/com/example/android_demo_kotlin/MainActivity.kt [hunk 2]
```diff
@@ -1,11 +1,23 @@
 package com.example.android_demo_kotlin
 
+import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import kotlinx.android.synthetic.main.activity_main.*
```
Comment (purvesh-d) [inline app/src/main/java/com/example/android_demo_kotlin/MainActivity.kt:7]: Remove * from import

**Review by purvesh-d — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/android_demo_kotlin/MainActivity.kt (hunk 2 above)
Comment (buddhdev-jay) [reply app/src/main/java/com/example/android_demo_kotlin/MainActivity.kt:7]: Changes Done and updated

**Pull request merged**